### PR TITLE
wicked: Add autoyast profile for opensuse wicked tests

### PR DESCRIPTION
**Pull request by cfconrad**
You can specify update repositories with csv via `AUTOYAST_ADD_REPOS`
variable.

- Related ticket: https://progress.opensuse.org/issues/66979
- Verification run: http://cfconrad-vm.qa.suse.de/tests/7488

### CODE REVIEW

File: data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      % for my $repo (split(/,/, $get_var->('AUTOYAST_ADD_REPOS',''))) {
```
Comment (asmorodskyi) [inline data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep:6]: @rwx788  WDYT ? is it right way to do it ?

File: data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  <software>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <products config:type="list">
+        <product>openSUSE</product>
+    </products>
+    <patterns config:type="list">
+      <pattern>base</pattern>
+    </patterns>
+    <packages config:type="list">
+      <package>zypper</package>
```
Comment (asmorodskyi) [inline data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep:35]: there is no zypper by default ?

File: data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep (hunk 1 above)
Comment (rwx788) [reply data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep:6]: TBH, we have other variables to control add-ons, actually too many of them, like `SCC_ADDONS`, just `ADDONS`. And for SLE 15 using urls will work only with full medium. Other than that, it's fine.

File: data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep (hunk 2 above)
Comment (cfconrad) [reply data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep:35]: Don't know. But this is what I really need :)

File: data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep (hunk 1 above)
Comment (cfconrad) [reply data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep:6]: Yes thanks for the hint. That was my expression as well. And none of them really fits my need.

There is actually also that `MAINT_TEST_REPO` variable which almost doing the same. But I didn't like the approach that we add none common variables into that template replacement logic if it isn't needed.

I also see quite often REPO_1, REPO_2 ... variables. Maybe we could use these too... 

For now I would just keep my logic. It doesn't influence any other and for me, this is what the template logic is for.

**Comment by asmorodskyi** [comment]
and why file has `.xml.ep` instead `.xml` ?

**Comment by cfconrad** [comment]
> and why file has `.xml.ep` instead `.xml` ?

because we are using the template mechanize from `lib/autoyast.pm`

**Pull request merged**